sdk/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb

```diff
@@ -630,6 +630,64 @@
     "hd_job"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4.3. Automatic hyperparameter sweeping for your AutoML models for computer vision tasks (Automode)\n",
+    "\n",
+    "When using AutoML for Images, we can perform an automatic hyperparameter sweep to find the optimal model (Automode feature). The feature will choose a model architecture and values for the learning_rate, number_of_epochs, layers_to_freeze, etc. based on the number of runs. There is no need to specify the hyperparameter space, sampling method or early termination policy.\n",
+    "\n",
+    "The Automode feature is triggered by setting `max_trials` to a value greater than 1 in limits."
```

Comment:
same comments as above on all the other notebooks.

Reply:
Addressed.